src/server/translate_cache.ts

```diff
@@ -26,12 +26,26 @@ import {Model, ModelMaterializer, Runtime} from '@malloydata/malloy';
 import {TextDocument} from 'vscode-languageserver-textdocument';
 
 import {ConnectionManager} from '../common/connection_manager';
-import {CellData} from '../common/types';
+import {BuildModelRequest, BuildModelResponse, CellData} from '../common/types';
 
 export class TranslateCache implements TranslateCache {
   cache = new Map<string, {model: Model; version: number}>();
 
-  constructor(private connection: Connection) {}
+  constructor(
+    private documents: TextDocuments<TextDocument>,
+    private connection: Connection,
+    private connectionManager: ConnectionManager
+  ) {
+    connection.onRequest(
+      'malloy/fetchModel',
+      async (event: BuildModelRequest): Promise<BuildModelResponse> => {
+        const model = await this.translateWithCache(event.uri, event.version);
+        return {
+          json_explores: model.explores.map(explore => explore.toJsonString()),
```

Comment:
These don't need to be stringified, the IPC will serialize them.

Reply:
Removed the `toJsonString()` and `fromJsonString()` and to return a JSON object directly.